plugins/window_size/linux/include/window_size/window_size_plugin.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef PLUGINS_WINDOW_SIZE_LINUX_INCLUDE_WINDOW_SIZE_WINDOW_SIZE_PLUGIN_H_
+#define PLUGINS_WINDOW_SIZE_LINUX_INCLUDE_WINDOW_SIZE_WINDOW_SIZE_PLUGIN_H_
+
+// A plugin to allow resizing the window.
+
+#include <flutter_plugin_registrar.h>
+
+#ifdef WINDOW_SIZE_PLUGIN_IMPL
```

Comment:
Is this idiom used across all the plugins? It might be nice documenting it somewhere. It's not immediately obvious at this site where WINDOW_SIZE_PLUGIN_IMPL is expected to be defined.